platform-coop-toolkit/coop-library-framework
# Add input validation for required fields (resolves #15)

## Description

Resolves #15.

## Steps to test

1. Try to edit or submit a resource, leaving the permanent link and publication date fields blank.

**Expected behavior:** Errors are clearly communicated (including via screen readers).

## Additional information

First pass. Related to https://core.trac.wordpress.org/ticket/47505.

## Related issues

- #2 
- #15
